aten/src/ATen/core/dispatch/Dispatcher.cpp

```diff
@@ -32,6 +32,7 @@ Dispatcher::Dispatcher()
 : operators_()
 , operatorLookupTable_()
 , backendFallbackKernels_()
+, backendFallbackNonFallthroughSet_(DispatchKeySet::FULL)
```

Comment:
This name and the `backend_nonfallthrough_mask` name above could probably be unified into one name to make clear that those are the same thing. I like this name here better than the `backend_nonfallthrough_mask` above but something like `backendsWithoutFallthrough` is probably even shorter and clearer.